public/src/components/common/list_btngrp.jsx

```diff
@@ -7,9 +7,10 @@ const ListBtnGroup = (props) => {
     <ButtonGroup vertical>
       {
         buttons.map((button) => {
-          const { style, text, handler, size, key } = button;
+          const { style, text, handler, size, key, disabled } = button;
```

Comment:
Is this affecting any other button groups?

Reply:
It's not affecting the other ones in a way that they don't need the `disabled` attributes so it's silently ignored. I needed this to block button clicks for unavailable transitions depending on the current campaign status.